### PR TITLE
ncm-modprobe: merge config-{common,rpm} into config.pan

### PR DESCRIPTION
**Pull request by ajf8**

### CODE REVIEW

**Comment by stdweird** [comment]
LGTM

**Comment by ajf8** [comment]
Thanks, just amended the commit msg as it said ncm-pam instead of ncm-modprobe.

File: ncm-modprobe/src/main/pan/components/modprobe/config.pan [hunk 1]
```diff
@@ -5,5 +5,17 @@
...
+include 'components/${project.artifactId}/schema';
+
+# Package to install
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+# Set prefix to root of component configuration.
+prefix '/software/components/${project.artifactId}';
+
+#'version' = '${project.version}';
+#'package' = 'NCM::Component';
```
Comment (ned21) [inline ncm-modprobe/src/main/pan/components/modprobe/config.pan]: Delete this line?

**Comment by stdweird** [comment]
LGTM
@ajf8 update branch

**Comment by jrha** [comment]
@stdweird what is this "update branch"? It seems to create merge commits, which are messy.

**Comment by ajf8** [comment]
Sure, I've done a rebase.

**Pull request merged**